src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void draw() {
+    }
+}
```

Comment:
Don't use abstract classes to set behavior for classes. Consider whether `Figure` should be an abstract class or an interface based on its purpose in your design. Abstract classes are typically used when you have a base class that should not be instantiated on its own, but has some common behavior or state for subclasses. If `Figure` is only meant to define a contract for its subclasses without providing any implementation, it should be an interface.